packages/api/package.json

```diff
@@ -26,6 +26,7 @@
     "@forge/validators": "workspace:*",
     "@stripe/stripe-js": "^5.2.0",
     "@trpc/server": "catalog:",
+    "axios": "^1.7.9",
```

Comment:
We don't need to use axios. The built-in `fetch` is sufficient for what we need.

Reply:
Used the discord client for type safety instead